authentication/migrations/0002_auto_20190905_1304.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.1 on 2019-09-05 13:04
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Hey @Niyitangasam, I'm not very sure but I think the team agreed on merging migrations so that we have only one migrations file per app. @Victoradukwu kindly clarify.

Reply:
Yes @wangonya, I worked on that.